BeeSwift/HealthKitConfigViewController.swift

```diff
@@ -92,14 +115,16 @@ class HealthKitConfigViewController: UIViewController {
         }
     }
     
-    func fetchGoals() {
+    @objc func fetchGoals() {
         MBProgressHUD.showAdded(to: self.view, animated: true)
         CurrentUserManager.sharedManager.fetchGoals(success: { (goals) in
             MBProgressHUD.hideAllHUDs(for: self.view, animated: true)
             self.goals = goals
+            self.tableView.refreshControl?.endRefreshing()
```

Comment:
a tweak would be addressing having both MBProgressHUD and refreshControl visible